third_party/move/tools/move-cli/tests/build_tests/dependency_chain/args.v2_exp

```diff
@@ -0,0 +1,10 @@
+Command `build -v`:
+INCLUDING DEPENDENCY Bar
+INCLUDING DEPENDENCY Foo
+BUILDING A
+[0m[1m[38;5;11mwarning[0m[1m: unknown attribute[0m
```

Comment:
I can see some weird chars in this exp file (but not in the corresponding v1 exp file)? Could you please double check these? Asking because I have not seen these in other exp files so far.

Reply:
![Screenshot 2024-04-23 at 9 55 50 AM](https://github.com/aptos-labs/aptos-core/assets/828050/196bb5f6-9f86-4ec1-b10d-c36298f2b374)

A screenshot of what I am seeing in github.dev if that is helpful.

Reply:
Those are ANSI color codes.  It's useful for humans, but should we not record it in test output files?

Reply:
Ok, figured out that it was already being done for move-compiler, added a new mechanism for others.